### DISCUSSION TITLE
Display when 7tv emotes are added and automatically refresh

### DISCUSSION THREAD

**Discussion by Velegand** [SevenTV/chatterino7#60] [Ideas]
The web extension of 7tv displays when emotes are added and does not require users to refresh after an emote is added
![UVKpTWW50Q](https://user-images.githubusercontent.com/69394829/144524848-4090c8fc-bec4-4c06-ae7a-c34b48e8b632.gif)

**Comment by Nerixyz** [comment]
#75 adds support for this.

**Comment by Nerixyz** [comment]
This has been added in both Chatterino and Chatterino7.
